x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/analytics_list/expanded_row.tsx

```diff
@@ -158,7 +159,11 @@ export const ExpandedRow: FC<Props> = ({ item }) => {
       defaultMessage: 'State',
     }),
     items: Object.entries(stateValues).map(s => {
-      return { title: s[0].toString(), description: getItemDescription(s[1]) };
+      if (s[0].toString() === 'state') {
+        return { title: s[0].toString(), description: getTaskStateBadge(getItemDescription(s[1])) };
+      } else {
+        return { title: s[0].toString(), description: getItemDescription(s[1]) };
+      }
```

Comment:
nit:
```
items: Object.entries(stateValues).map(([stateKey, stateValue]) => {
    if (stateKey.toString() === 'state') {
        return { title: stateKey.toString(), description: getTaskStateBadge(getItemDescription(stateValue)) };
     }
     return { title: stateKey.toString(), description: getItemDescription(stateValue) }; 
})
```

Reply:
Done in https://github.com/elastic/kibana/pull/60249/commits/d8926e25a3a4e1c7e3dd20a6342c15cba035bfc5